CONTRIBUTING.md

```diff
@@ -109,11 +103,11 @@ Since the deploy preview does not work on Draft PRs, follow these steps to previ
 
 4. Start the local instance of the tutorial build:
 
-    ```sh
-    $ npm run serve
-    ```
+   ```sh
+   $ npm run serve
+   ```
```

Comment:
I am confused why there are still 3 spaces on this indentation. I thought your commits reverted that.

Reply:
Done!